internal/config/logger.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"fmt"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+func ConfigLogger(config *Config) error {
+	level, err := logger.ParseLevel(config.Logger.Level)
+	if err != nil {
+		return fmt.Errorf("error parsing log level: %w", err)
+	}
+	logger.SetLevel(level)
+	logger.Info("Logger configured successfully.")
```

Comment:
I always wonder: Shouldn't this log be in main.go after logger has been initialized? It's like if I was to use this in my application, there was no way to disable this log.

Reply:
Hmm, you're actually right, I would also expect this log in the main.go file